src/InboundServer/api.yaml

```diff
@@ -4453,6 +4456,4 @@ components:
           $ref: '#/components/schemas/ExtensionList'
           description: 'Optional extension, specific to deployment.'
       required:
-        - transferState
-
-
+        - transferState
```

Comment:
Please add a new line at the end of the file

Reply:
Line added

Reply:
line added